_posts/2019-07-22-aws-event-bridge.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: AWS Event Bridge Introduced
```

Comment:
maybe "Introducing AWS Event Bridge"

Reply:
I was speaking in the past tense, since it was already announced by AWS, and that it's a post about what they've already done. I do think the title could be better though.